src/fstests/fstests.jinja2

```diff
@@ -40,7 +45,7 @@ class Job(BaseJob):
 
     def _get_xml_from_vm(self, kdir):
         try:
-            result = subprocess.run([f"debugfs -R \"dump ext4/results-4k/results.xml {kdir}/results.xml\" /home/kernelci/alex/fstests-poc/fstests/disks/vdg"], check=True, shell=True)
+            result = subprocess.run([f"debugfs -R \"dump ext4/results-4k/results.xml {kdir}/results.xml\" {xfstests_bld_path}/run-fstests/disks/vdg"], check=True, shell=True)
```

Comment:
Alright, this should be part of the first commit as there's no value in first adding the hard-coded path and then removing it - it's not part of a logic series of development steps.